Description: Upstream changes introduced in version 3.7-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 premake (3.7-1) unstable; urgency=low
 .
   * Initial release. (Closes: #262864)
 .
 The person named in the Author field signed this changelog entry.
Author: James Goppert <james.goppert@gmail.com>
Bug-Debian: http://bugs.debian.org/262864

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- premake-3.7.orig/Makefile
+++ premake-3.7/Makefile
@@ -15,7 +15,7 @@ all: Premake
 
 Makefile: premake.lua Src/premake.lua
 	@echo ==== Regenerating Makefiles ====
-	@premake --file $^ --clean --target gnu
+	@premake --file $^ --target gnu
 
 Premake:
 	@echo ==== Building Premake ====
